feature/agenda/presentation/src/main/java/com/sergius/agenda/presentation/agendaitem/AgendaItemDetailsRoot.kt

```diff
@@ -148,15 +151,37 @@ private fun TaskDetails(
                     onAction = onAction,
                     itemType = itemType.capitalize(),
                 )
-                TaskyDivider()
 
-                TaskDateTime(
-                    onAction = onAction,
-                    showTimerDialog = state.showTimerDialog,
-                    timePickerState = state.timePickerState,
-                    showDateDialog = state.showDateDialog,
-                    datePickerState = state.datePickerState
-                )
+                if (AgendaItemType.EVENT == itemType) {
```

Comment:
Suggestion 🙂 this block looks like a good candidate for its own composable section IMO - could help keep the parent cleaner.